packages/angular/src/stories/button/button.stories.ts

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc.
+ * All Rights ReserveThis software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { moduleMetadata } from '@storybook/angular';
+import { withDesign } from 'storybook-addon-designs';
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { boolean, select } from '@storybook/addon-knobs';
+import { ClrButtonModule, ClrLoadingState } from '@clr/angular';
```

Comment:
Generally we just suggest importing `ClarityModule` instead of isolating modules. I suggest we do the same with all stories, unless we see a perf problem.

Reply:
Done.